docs/proposals/n000-proposals.md

```diff
@@ -0,0 +1,92 @@
+---
+np: 000
+title: Proposals
+status: Draft
+type: Meta
+author: George Antoniadis <george@noodles.gr>
+created: 2020-12-20
+---
+
+# np000 - Proposals
+
+## Simple Summary
+
+A formal approach to proposing new features or updates for nimona.
+
+## Problem Statement
+
+Until now nimona's spec and protocol was very fluid in order to allow us to
+quickly iterate without fear or braking things. This has resulted in a lack of
+specifications and concrete documentation.
+
+As we are moving closer to a more solid protocol we'll need to spend more time
+designing updates in order to make sure everything is backwards compatible and
+does not break apps.
+
+In addition to this, we need a good way to help newcomers understand the various
+parts of nimona and the rationale behind some of the design decisions.
+
+## Proposal
+
+Instead of trying to maintain specification docs, create distinct and standalone
+proposals for each new feature or update.
+
+Documentation will serve as a TL;DR and attempt to guide the reader through the
+various proposals.
+
+The proposal process itself is influenced by
+[golang's](https://github.com/golang/proposal#design-documents) and
+[ethereum's](https://eips.ethereum.org/EIPS/eip-1)
+proposal processes.
+
+### Index example
+
+* Objects
+  * np001 - Hinted objects
+  * np003 - Consistent hashing
+  * np007 - Streams
+  * np008 - Feeds
+* Network
+  * np002 - Message based network
+  * np004 - Enforce mTLS
+  * np006 - Object relaying
+* Discovery
+  * np005 - Hyperspace
+
+### Proposal template
+
+Proposals should be written in markdown format and should be wrapped around the
+80 column mark.
+
+A [template](np-template.md) exists that provides the basic structure for the
+proposal.
+
+### Proposal Review
+
+A group of contributors will be holding “proposal review meetings” to review
+pending proposals.
+
+The principal goal of the review meeting is to make sure that proposals are
+receiving attention from the right people, by cc'ing relevant developers,
+raising important questions, pinging lapsed discussions, and generally trying to
+guide discussion toward agreement about the outcome. The discussion itself is
+expected to happen on the proposal's discussion page.
```

Comment:
I am a bit confused as to if this is a video call or comments on the PR for the proposal.

Reply:
The way I see this is a meeting that the notes are made on the proposal pr. Is this right?

Reply:
I guess maybe this is intentionally vague as to not be restrictive? But maybe some suggestions could be mentioned like where/how this happens, on a PR adding the proposal document, etc.

Reply:
reworded this section a bit.
all discussion should be happening on the discussions page.
meeting is to make sure that we are dealing with proposals.